### DISCUSSION TITLE
'&' in windows filename splits in argument list parsing

### DISCUSSION THREAD

**Discussion by mrmsbarnes** [Breakthrough/PySceneDetect#311] [Troubleshooting]
When I do
>scenedetect -i 1+2.avi detect-content
I get one scene break at 00:00:08.267
When I do
>scenedetect -i 1&2.avi detect-content
scenedetect says:
Error: Missing command.
'2.avi' is not recognized as an
operable program or batch file.

The two files are identical except for file name. I had to append ".mp4" to each file name to get your bug tracker to accept them.
And, Yes, scenedetect (or at least ffmpeg) does handle .avi files.

https://user-images.githubusercontent.com/8269732/210949046-2cc96668-54b9-456c-91a1-a1d2acd72b96.mp4

https://user-images.githubusercontent.com/8269732/210949066-cb62ef84-6baa-4bc1-ac19-b52e499169dd.mp4

**Comment by mrmsbarnes** [comment]
also I notice the bad named file is noted as 1.2.avi.mp4 instead of 1<amp>2.avi.mp4 in files attached. Evidently ampersands are way too special to github to be allowed in filenames. Someone should tell Microsoft.

**Comment by mrmsbarnes** [comment]
Comment to my comment to my submitted bug. Sorry I don't know how to convey a literal ampersand in this forum.

**Comment by Breakthrough** [comment]
The `&` symbol has special meaning and should be "escaped" by prefixing it with a `^`. So in order to process a file named `1&2.avi` you would need to run:

    scenedetect -i 1^&2.avi detect-content

Double quotes might also work in some cases, e.g. `scenedetect -i "1&2.avi" detect-content`.  Hope this helps, see https://ss64.com/nt/syntax-esc.html for details.  Thanks!

**Comment by mrmsbarnes** [comment]
Thank you. I will test this shortly. Good to know ^ is escapement in this context. \\;-)
